### ISSUE TITLE
could not determine kind of name for C.PANGO_ATTR_INSERT_HYPHENS

### ISSUE THREAD

**Issue by jstaf** [jstaf/onedriver#298]
Ran into this issue while building a release for EL8 and Debian 10

```
+ go build -v -mod=vendor '-ldflags=-X github.com/jstaf/onedriver/cmd/common.commit=b96e638d44799ba467dc86b96eeb3cf2b50beb64' ./cmd/onedriver
golang.org/x/sys/internal/unsafeheader
github.com/gotk3/gotk3/internal/slab
github.com/hanwen/go-fuse/v2/splice
github.com/coreos/go-systemd/v22/unit
golang.org/x/sys/unix
github.com/imdario/mergo
github.com/jstaf/onedriver/fs/graph/quickxorhash
github.com/rs/zerolog/internal/json
github.com/rs/zerolog
github.com/hanwen/go-fuse/v2/fuse
github.com/rs/zerolog/log
github.com/jstaf/onedriver/fs/graph
github.com/gotk3/gotk3/internal/callback
github.com/gotk3/gotk3/internal/closure
github.com/gotk3/gotk3/glib
gopkg.in/yaml.v3
go.etcd.io/bbolt
github.com/jstaf/onedriver/fs
github.com/spf13/pflag
github.com/gotk3/gotk3/cairo
github.com/gotk3/gotk3/pango
github.com/gotk3/gotk3/gdk
# github.com/gotk3/gotk3/pango
vendor/github.com/gotk3/gotk3/pango/pango-attributes_since_1_44.go:10:33: could not determine kind of name for C.PANGO_ATTR_INSERT_HYPHENS
vendor/github.com/gotk3/gotk3/pango/pango-attributes_since_1_44.go:14:7: could not determine kind of name for C.pango_attr_insert_hyphens_new
error: Bad exit status from /var/tmp/rpm-tmp.SGuKSQ (%build)


RPM build errors:
    Bad exit status from /var/tmp/rpm-tmp.SGuKSQ (%build)
```

**Comment by jstaf** [comment]
https://github.com/gotk3/gotk3/issues/693 seems to be the issue.

**Comment by jstaf** [comment]
Fixed by #300
